tests/phpunit/Mutator/DefinitionTest.php

```diff
@@ -74,4 +79,37 @@ public function valuesProvider(): iterable
             'The diff',
         ];
     }
+
+    /**
+     * @dataProvider mutatorsProvider
+     */
+    public function test_it_must_define_remedies(
+        Mutator $mutator
+    ): void {
+        $this->assertNotNull(
+            $mutator::getDefinition()->getRemedies(),
+            sprintf(
+                'Definition of [%s] must provide remedies.',
+                $mutator->getName(),
+            ),
+        );
+    }
+
+    public function mutatorsProvider(): iterable
+    {
+        $mutatorFactory = SingletonContainer::getContainer()->getMutatorFactory();
```

Comment:
The reason for this inline call the the container is because providers run before `setUp` methods, in which case a call to something like `$this->mutatorFactory` would be `null`.